src/resources/Fields/Fields.ts

```diff
@@ -39,7 +39,14 @@ export default class Field extends Resource {
         return this.api.put(`${Field.baseUrl}/${fieldId}`, options);
     }
 
+    /**
+     * @deprecated list(params?: ListFieldsParams) is kept for backward compatibility, you should now use `search(params?: FieldListingOptions)`.
+     */
```

Comment:
Do you plan to bin it tho?

Reply:
Do you mean remove it? It's still supported from the field service as far as I know.

Reply:
Yea meant that, but OK 👍